python/pyspark/mllib/tests.py

```diff
@@ -75,22 +75,23 @@
     # No SciPy, but that's okay, we'll skip those tests
     pass
 
-ser = PickleSerializer()
```

Comment:
I ask because it looks like this change conflicted with a recently-modified test, causing the test to break:

```python
======================================================================
ERROR [7.323s]: test_als_ratings_id_long_error (pyspark.mllib.tests.ALSTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/workspace/SparkPullRequestBuilder/python/pyspark/mllib/tests.py", line 1571, in test_als_ratings_id_long_error
    self.assertRaises(Py4JJavaError, self.sc._jvm.SerDe.loads, bytearray(ser.dumps(r)))
NameError: global name 'ser' is not defined

======================================================================
ERROR [0.779s]: test_als_ratings_serialize (pyspark.mllib.tests.ALSTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/workspace/SparkPullRequestBuilder/python/pyspark/mllib/tests.py", line 1562, in test_als_ratings_serialize
    jr = self.sc._jvm.SerDe.loads(bytearray(ser.dumps(r)))
NameError: global name 'ser' is not defined
```